internal/analysis/document_symbols.go

```diff
@@ -31,8 +33,15 @@ func (r *CheckResult) GetSymbols() []DocumentSymbol {
 			Range:          v.Name.Range,
 			SelectionRange: v.Name.Range,
 		})
-
 	}
 
+	slices.SortFunc(symbols, func(a, b DocumentSymbol) int {
+		if a.Range.Start.GtEq(b.Range.Start) {
```

Comment:
not stable sort ?

Reply:
the order doesn't matter match. In fact, they weren't even sorted initially - it's just more handy to have a deterministic way to be outputted for testing purposes (they are taken from a map, whose order is not reliable)

Reply:
(and for this reason, stable sort wouldn't be "better" than regular sorting)